alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/main/resources/application-default.yaml

```diff
@@ -1,5 +1,8 @@
 queue:
   engineRequestQueue: ${TRANSFORM_ENGINE_REQUEST_QUEUE:org.alfresco.transform.engine.libreoffice.acs}
 transform:
-  config:
-    location: classpath:libreoffice_engine_config.json
\ No newline at end of file
+  core:
+    config:
+      location: classpath:libreoffice_engine_config.json
+    libreoffice:
```

Comment:
It is not enough to just change the transform.core.config.location property in the properties file, This way you have just unset it. You need to update it in the TransformRegistryImpl, otherwise it will just return the empty default config. Keep in mind that all other transformers have the same property too, that needs updating.

Reply:
We need to make sure the transform configs are the same before and after these changes.

Reply:
Good catch. I was meant to that.